src/dmd/backend/cdef.d

```diff
@@ -49,7 +49,7 @@ version (XVERSION)
     enum TARGET_FREEBSD = xversion!`FreeBSD`;
     enum TARGET_OPENBSD = xversion!`OpenBSD`;
     enum TARGET_SOLARIS = xversion!`Solaris`;
-    enum TARGET_WINDOS  = xversion!`Windows`;
+    @trusted bool TARGET_WINDOS() { return (config.exe & EX_windos) != 0; }
```

Comment:
I left this as a function to minimise breakage...

Reply:
Actually, it breaks things, as the dmc++ front end still does `static if (TARGET_WINDOS)`. Please revert this line.

Reply:
done